go.mod

```diff
@@ -1,34 +1,22 @@
 module github.com/RHsyseng/operator-utils
 
-go 1.13
+go 1.16
 
 require (
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-logr/logr v0.2.1
-	github.com/go-logr/zapr v0.2.0 // indirect
+	github.com/go-logr/logr v0.4.0
 	github.com/go-openapi/spec v0.19.9
 	github.com/go-openapi/strfmt v0.19.5
 	github.com/go-openapi/validate v0.19.11
 	github.com/googleapis/gnostic v0.5.1
-	github.com/openshift/api v0.0.0-20200930075302-db52bc4ef99f
-	github.com/openshift/client-go v0.0.0-20200929181438-91d71ef2122c
+	github.com/openshift/api v0.0.0-20210105115604-44119421ec6b
+	github.com/openshift/client-go v0.0.0-20210112165513-ebc401615f47
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.42.1
-	github.com/stretchr/testify v1.6.1
-	go.uber.org/zap v1.16.0
-	k8s.io/api v0.19.0
-	k8s.io/apimachinery v0.19.0
-	k8s.io/client-go v0.19.0
-	sigs.k8s.io/controller-runtime v0.6.3
-)
-
-replace (
-	// OpenShift release-4.6
-	github.com/openshift/api => github.com/openshift/api v0.0.0-20200930075302-db52bc4ef99f
-	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200929181438-91d71ef2122c
-
-	// Pinned to kubernetes-1.19.0
-	k8s.io/api => k8s.io/api v0.19.0
-	k8s.io/apimachinery => k8s.io/apimachinery v0.19.0
-	k8s.io/client-go => k8s.io/client-go v0.19.0 // Required by prometheus-operator
```

Comment:
is this no longer necessary with go 1.16? i haven't tested but saw something about this listed in release notes - https://go.dev/blog/go116-module-changes

Reply:
This is not related to go 1.16. After upgrade of api versions, now every imported dependency is compatible with k8s api v0.20.4. There is no need of `replace`.